<launch>
    <node pkg="aama_sim" name="sim_controller" type="simulation_controller.py" output="screen"/>
    <node pkg="aama_sim" name="sensor_controller" type="sensor_controller.py" output="screen"/>
</launch>